worker/combined/worker.go

```diff
@@ -0,0 +1,35 @@
+package combined
+
+import (
+	"fmt"
+
+	"github.com/0xSplits/workit/worker/parallel"
+	"github.com/0xSplits/workit/worker/sequence"
+	"github.com/xh3b4sd/tracer"
+)
+
+type Config struct {
+	Par *parallel.Worker
+	Seq *sequence.Worker
+}
+
+// Worker is a simple wrapper that combines the different worker engines in a
+// single daemon interface.
+type Worker struct {
+	par *parallel.Worker
+	seq *sequence.Worker
+}
```

Comment:
We are using this now in Kayron because of different jobs that the operator now takes on as well, e.g. container image cleanup.